networkx/algorithms/bipartite/edgelist.py

```diff
@@ -243,7 +243,7 @@ def parse_edgelist(
                 edgedata = dict(literal_eval(" ".join(d)))
             except BaseException as err:
                 raise TypeError(
-                    f"Failed to convert edge data ({d})" f"to dictionary."
+                    f"Failed to convert edge data ({d}) to dictionary."
```

Comment:
This is the only actual change to a string. I added a space--hence what pushed me to make a PR ;)